src/internal/matcher/nearly_eq.rs

```diff
@@ -1,29 +1,74 @@
+use crate::internal::argument::Argument;
 use crate::internal::fmt::MaybeDebug;
 use crate::internal::fmt::MaybeDebugWrapper;
 use crate::internal::ArgumentMatcher;
 use nameof::name_of;
 use nearly_eq::NearlyEq;
 use std::fmt::{self, Debug, Display};
 
-/// Creates a new `ArgumentMatcher` that matches against values using [`NearlyEq`]
-pub fn nearly_eq<T, U>(value: T) -> NearlyEqArgumentMatcher<T, U>
-where
-    T: NearlyEq<T, U> + MaybeDebug,
-    U: MaybeDebug,
-{
-    NearlyEqArgumentMatcher {
-        value,
-        accuracy: T::eps(),
+impl Argument {
+    /// Creates an argument matcher that matches values using [`NearlyEq`].
+    /// Uses the default epsilon value defined by [`NearlyEq`].
+    ///
+    /// # Examples
+    /// ```
+    /// use mockiato::mockable;
+    ///
+    /// #[cfg_attr(test, mockable)]
+    /// trait FloatFormatter {
+    ///     fn format_float(&self, value: f64) -> String;
+    /// }
+    ///
+    /// let expected_result = String::from("0.3");
+    /// let mut formatter = FloatFormatterMock::new();
+    /// formatter
+    ///     .expect_format_float(|a| a.nearly_eq(0.3))
```

Comment:
Again, I feel like using `arg` is nicer for beginners than `a`. Check this globally if you agree.

Reply:
Addressed in f22cd76